src/components/Person/Person.jsx

```diff
@@ -1 +1,48 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    <p className="Person__age">
+      {person.age && (
+        <p>{`I am ${person.age}`}</p>
+      )}
+    </p>
+    <p className="Person__partner">
+      {!person.isMarried ? (
+        <p>I am not married</p>
+      ) : (
+        <p>
+          {`${person.partnerName} is my ${person.sex === 'm'
+            ? 'wife'
+            : 'husband'
+          }`}
```

Comment:
Consider moving this logic into a function. The double ternary operator affects the readability and maintainability of your code